### PR TITLE
FEATURE: Better interoperability with assign plugin.

### PR DESCRIPTION
**Pull request by nbianca**

### CODE REVIEW

File: config/settings.yml [hunk 1]
```diff
@@ -16,3 +16,5 @@ plugins:
   code_review_allow_self_approval:
     client: true
     default: false
+  code_review_auto_assign_on_followup: true
+  code_review_auto_unassign_on_approve: true
```
Comment (SamSaffron) [inline config/settings.yml]: oops, looks like we are missing 2 translations here.

**Comment by SamSaffron** [comment]
I know this is somewhat odd to ask here, but I think there is long term benefit here with the assign plugin. 

Can you have it define "DiscourseEvent" cause then we can simply use then to assign / unassign. 

https://github.com/discourse/discourse/blob/37088c4221bbaa8de1d24309376da1740ce455a2/config/initializers/012-web_hook_events.rb#L9-L11

So then to fire it we would do something like:

```
DiscourseEvent.trigger(:assign_topic, topic, user, assigning_user) 
```

It opens the door to cleaner integration and much less fragile code long term.

**Comment by nbianca** [comment]
Added the events in discourse/discourse-assign#24.

**Comment by SamSaffron** [comment]
Sure lets try this out 🎊

**Pull request merged**